lighthouse-core/lib/stack-packs.js

```diff
@@ -5,8 +5,21 @@
  */
 'use strict';
 
-const stackPacks = require('../../stack-packs/index.js');
 const log = require('lighthouse-logger');
+const stackPacks = require('lighthouse-stack-packs');
+const i18n = require('./i18n/i18n.js');
+
+/**
+ * Resolve a module on web and node
+ * @param {string} module
+ */
+function resolve(module) {
+  if (!require.resolve) {
+    return `node_modules/${module}`;
+  }
+
+  return require.resolve(module);
+}
```

Comment:
Added a require.resolve to make sure we still do require.resolve in node land but we don't in web land.